docs/run/advanced-runtime-options.mdx

```diff
@@ -1,25 +1,68 @@
 ---
-title: Algorithm tuning options
-description: Common settings to use when tuning primitive options
+title: Advanced runtime options
```

Comment:
I was going to challenge if this should be called "Advanced"? I see Abby M suggested it.